src/client.cpp

```diff
@@ -275,7 +129,7 @@ void ADDON_Destroy()
 
 ADDON_STATUS SetStringSetting(const std::string &settingName, const char* settingValue, std::string &currentValue, ADDON_STATUS returnValueIfChanged)
 {
-  string strSettingValue = settingValue;
+  std::string strSettingValue = settingValue;
```

Comment:
Why is here still another set of Add*Setting functions???

Reply:
My bad